services/catarse/spec/models/project_spec.rb

```diff
@@ -718,6 +718,21 @@ def all_machine_states
     end
   end
 
+  describe 'update_service_slip_fee' do
+    context 'when project is flex or aon' do
+      let(:project) { create(:project, state: 'online', service_slip_fee: 0, mode: 'flex') }
+
+      slip_tax = CatarseSettings.get_without_cache(:catarse_slip_tax).to_i
+
+      before do
+        project.update(service_slip_fee: slip_tax)
+        project.valid?
+      end
+
+      it { expect(project.service_slip_fee).to eq(slip_tax) }
+    end
+  end
+
```

Comment:
**Sugestão**
Esse teste não está cobrindo se a funcionalidade de adicionar taxa está realmente funcionando pois o update que tu deu foi manual, enquanto isso é pra ser automático e também tu não cobriu a condição de quando ele for um projeto de assinaturas. Um teste mais seguro seria assim (os comentários são pra te explicar o que está acontecendo, não precisa usá-los no código):
```suggestion
  describe 'fill_service_slip_fee' do
    # Pego um numero aleatório
    let(:service_slip_fee) { Faker::Number.number(digits: 3) }
    
    # Defino o número aleatório como a taxa do boleto
    before { CatarseSettings[:catarse_slip_tax] = service_slip_fee }
    
    context 'when project is flex or aon' do
      # atenção pro %w[flex aon].sample, pois tu estava deixando fixo sempre para o flex,
      # se alguém algum dia tirar o aon do teste esse teu código não detectaria o erro.
      let(:project) { create(:project, state: 'online', service_slip_fee: 0, mode: %w[flex aon].sample) }
          
      # Testo que o projeto foi preenchido com a taxa configurada
      it { expect(project.service_slip_fee).to eq(service_slip_fee) }
    end
  end
  
  context 'when project is sub' do
      let(:project) { create(:project, state: 'online', service_slip_fee: 0, mode: 'sub') }
          
      # Testo que o projeto não foi preenchido com a taxa configurada
      it { expect(project.service_slip_fee).to be_zero }
    end
  end

```